core-bundle/src/Resources/contao/modules/ModuleSearch.php

```diff
@@ -313,13 +315,73 @@ protected function compile()
 					$objTemplate->hasContext = true;
 				}
 
+				$this->addImageToTemplateFromSearchResult($arrResult[$i], $objTemplate);
+
 				$this->Template->results .= $objTemplate->parse();
 			}
 
 			$this->Template->header = vsprintf($GLOBALS['TL_LANG']['MSC']['sResults'], array($from, $to, $count, $strKeywords));
 			$this->Template->duration = System::getFormattedNumber($query_endtime - $query_starttime, 3) . ' ' . $GLOBALS['TL_LANG']['MSC']['seconds'];
 		}
 	}
+
+	protected function addImageToTemplateFromSearchResult(array $result, Template $template): void
+	{
+		$template->hasImage = false;
+
+		if (!isset($result['meta']) || null == $result['meta'])
+		{
+			return;
+		}
+
+		$meta = json_decode($result['meta'], true);
+
+		// Reverse so the last element wins
+		$meta = array_reverse($meta);
+
+		foreach ($meta as $v)
+		{
+			if (!isset($v['https://schema.org/primaryImageOfPage']))
+			{
+				continue;
+			}
+
+			$figureBuilder = $this->getFigureBuilderIfResourceExists($v['https://schema.org/primaryImageOfPage']['contentUrl']);
```

Comment:
Leftover :slightly_smiling_face: 
```suggestion
```

Reply:
Yeah I've already fixed that locally but the CS fixer takes forever to run :D